BangazonAPI/Models/Department.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+	public class Class
+	{
```

Comment:
Let's make sure that we name our classes

Reply:
should be fixed now